subprojects/docs/src/docs/userguide/java_library_plugin.adoc

```diff
@@ -237,7 +237,10 @@ The role of each configuration is described in the following tables:
 [[sec:java_library_known_issues_compat]]
 === Compatibility with other plugins
 
-At the moment, the Java Library plugin is wired to behave correctly with the `java`, `groovy` and `kotlin` plugins. In Gradle 5.3 or earlier, some plugins, such as the Groovy plugin, may not behave correctly. In particular, if the Groovy plugin is used in addition to the `java-library` plugin, then consumers may not get the Groovy classes when they consume the library. To workaround this, you need to explicitly wire the Groovy compile dependency, like this:
+At the moment, the Java Library plugin is wired to behave correctly with the `java`, `groovy`, `scala` and `kotlin` plugins. In Gradle 5.3 or earlier, some plugins, such as the Groovy plugin, may not behave correctly.
```

Comment:
The way this is phrased is confusing. I think we should indicate that all is good with 5.6 and then have the info to do the wiring behind a link. Though this could wait for the documentation rework.

Reply:
I removed this section altogether. Given we have versioned docs on docs.gradle.org, this is redundant.